lib/kitchen/driver/oci.rb

```diff
@@ -153,15 +153,28 @@ def launch(state, inst)
       def create_and_attach_volumes(config, state, oci, api)
```

Comment:
There wasn't anything wrong here but the linter didn't like the method being too long so I split into to process the volumes and choose between volume types.